config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :books do
```

Comment:
Use the resources crud method read this https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions
like 
```ruby
resources :books, only: :show
```